packages/simulators/README.md

```diff
@@ -0,0 +1,40 @@
+## What is Simulators
+
+Simulators is a set of libraries for controlling mobile simulators/emulators. Currently there is support for iOS Simulators.
+
+## What can I use Simulators library for
+
+Currently only iOS is supported. There are two main classes `IosSimulator` and `IosSimulatorManager`. `IosSimulatorManager` can be used for accesing `IosSimulator` instances.
```

Comment:
nit: if you break up the lines into shorter ones, they will be more manageable even in editors that don't break up long lines for you.

Reply:
done